integration_tests/redis/redisHelpers.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-await-in-loop */
+import { createRedisClient } from '../../server/data/redisClient'
+
+const rateLimitKeyPattern = 'rateLimit:*'
+
+const clearRateLimits = async () => {
+  const client = createRedisClient()
+  await client.connect()
+
+  const keys = await client.keys(rateLimitKeyPattern)
+
+  if (keys.length > 0) {
+    await client.del(keys)
+  }
+
+  await client.quit()
+  return Promise.resolve(null)
+}
+
+const waitUntilRateLimitsExpire = async () => {
```

Comment:
Used to test rate limit expiry working: polls every 0.5s until no rate limit keys left in Redis. Limited to never try for more than 10 seconds (which won't be reached as Cypress default timeout is 6 seconds anyway)